defaults/gce_config.yaml

```diff
@@ -21,7 +21,7 @@ gce_n_local_ssd_disk_monitor: 0
 gce_instance_type_db: 'n2-highmem-8'
 gce_root_disk_type_db: 'pd-ssd'
 root_disk_size_db: 50
-gce_n_local_ssd_disk_db: 3
+gce_n_local_ssd_disk_db: 4
```

Comment:
Search shows couple more k8s related places, where the number of disks is 3. Should we update there as well? 
```
❯ grep -rn 'gce_n_local_ssd_disk_db: 3' ./
./test-cases/scylla-operator/longevity-scylla-operator-multidc-12h.yaml:43:gce_n_local_ssd_disk_db: 3
./test-cases/scylla-operator/longevity-scylla-operator-multidc-3h.yaml:43:gce_n_local_ssd_disk_db: 3
./defaults/k8s_gke_config.yaml:12:gce_n_local_ssd_disk_db: 3
./defaults/gce_config.yaml:24:gce_n_local_ssd_disk_db: 3
```

Reply:
if we change their instance type too, then yes

Reply:
I've updated them as well to 4